xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -386,8 +386,9 @@ COverlay* CRenderer::Convert(CDVDOverlaySSA* o, double pts)
   else
     position = 0.0;
   int changes = 0;
-  ASS_Image* images = o->m_libass->RenderImage(targetWidth, targetHeight, videoWidth, videoHeight, sourceWidth, sourceHeight,
-                                               pts, useMargin, position, &changes);
+  ASS_Image* images =
+      o->GetLibass()->RenderImage(targetWidth, targetHeight, videoWidth, videoHeight, sourceWidth,
```

Comment:
nullptr check needed?

Reply:
I checked the code and figured out what needed to be done in CDVDOverlayCodecSSA and CDVDSubtitlesLibass to avoid a null check here. It's not pretty and a lot more work, so I would just add the null check.

Reply:
I assume this is only a matter of returning early in the definition of the method right?

```
COverlay* CRenderer::Convert(CDVDOverlaySSA* o, double pts)
{
    if (!o->GetLibass)
      return
....
```

In the call to convert `nullptr` is checked:

https://github.com/xbmc/xbmc/blob/5f41658c047b185f87a44a3647d78b590d3b431a/xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.cpp#L162-L163